docs/modules/help/partials/faq_ticket_detail.adoc

```diff
@@ -0,0 +1,3 @@
+=== Why can't I close some tickets?
+
+Some tickets might be in such a state that can't be closed. Only new tickets, tickets in "backlog" and in "customer review" status can be closed. If you need to close such a ticket, please write a comment requesting to change the state.
```

Comment:
I haven't rechecked the states which allow closing, we might need to make sure that this actually is this list.

Reply:
I copied this text from the tooltip that appears when the "Close" button is dimmed and not clickable.